src/studio/src/designer/frontend/app-development/features/administration/components/SideMenuContent.tsx

```diff
@@ -112,7 +112,8 @@ const SideMenuContent = (props: ISideMenuContent): JSX.Element => {
         id='reset-repo-button'
         btnText={getLanguageFromKey('administration.reset_repo_button', props.language)}
         onClickFunction={onClickResetRepo}
-      />
+      /><br/>
+      <a href={`/designer/api/v1/repositories/${(window as any).org}/${(window as any).app}/contents.zip`}>{getLanguageFromKey('administration.download_repo', props.language)}</a>
```

Comment:
I'm not sure why the link is placed between the reset-button and it's modal?
Also, casting to any twice within the markup looks a little messy.
Could you maybe move the url to one of the urlHelper-files for consistency?

Reply:
Totally agree. I just wanted to see how it would look and feel when in use. If you want this functionality I'll argue for hiding this behind a modal (like the reset changes) to give an extra protection for unintended activation.

Reply:
A Modal will also be a great place to explain how to extract the archive to a local git repo and commit the changes.